env.local.php

```diff
@@ -1,47 +0,0 @@
-<?php
```

Comment:
Oh we missed it. Actually this needs to be renamed to run on local environment, as required by laravel framework. So we renamed it to .env.local.php . So it got deleted. Thats a mistake shouldn't be like that.